src/test/resources/org/broadinstitute/hellbender/tools/exome/allelic-pon-test-pon-counts-normal.tsv

```diff
@@ -0,0 +1,1001 @@
+1	206	1126	1347	50
+1	207	1251	1170	49
+1	208	1290	1182	50
+1	209	1333	1130	49
+1	210	1274	1254	50
+1	211	1349	1233	50
+1	212	1282	1201	50
+1	213	1299	1189	50
+1	214	1230	1313	50
+1	215	1321	1210	50
+1	216	1288	1118	50
+1	217	1327	1203	50
+1	218	1270	1231	50
+1	219	1286	1184	50
+1	220	1301	1156	50
+1	221	1458	1050	50
+1	222	1190	1267	50
+1	223	1435	1048	50
+1	224	1325	1235	50
+1	225	1152	1386	50
+1	226	1177	1219	50
+1	227	1326	1236	50
+1	228	1371	1057	50
+1	229	1216	1239	50
+1	230	1312	1190	50
+1	231	1148	1323	50
+1	232	1352	1080	50
+1	233	1288	1175	50
+1	234	1230	1196	49
+1	235	1138	1295	50
+1	236	1468	1129	50
+1	237	1285	1254	50
+1	238	1333	1082	50
+1	239	1246	1210	50
+1	240	1257	1234	50
+1	241	1232	1234	50
+1	242	1391	1229	50
+1	243	1298	1149	50
+1	244	1352	1170	50
+1	245	1138	1352	50
+1	246	1429	1101	50
+1	247	1401	1103	50
+1	248	1318	1210	50
+1	249	1466	996	50
+1	250	1494	1115	50
+1	251	1328	1203	50
+1	252	1405	1059	50
+1	253	1332	1160	50
+1	254	1195	1327	50
+1	255	1260	1253	50
+1	256	1331	1159	50
+1	257	1404	1061	50
+1	258	1167	1283	50
+1	259	1200	1345	50
+1	260	1366	1202	50
+1	261	1106	1329	50
+1	262	1328	1232	50
+1	263	1330	1118	50
+1	264	1194	1265	50
+1	265	1303	1231	50
+1	266	1464	1133	50
+1	267	1294	1264	50
+1	268	1241	1335	50
+1	269	1298	1193	50
+1	270	1208	1292	50
+1	271	1537	1059	50
+1	272	1311	1238	50
+1	273	1224	1292	50
+1	274	1305	1232	50
+1	275	1309	1192	50
+1	276	1257	1263	50
+1	277	1241	1272	50
+1	278	1270	1213	50
+1	279	1144	1324	50
+1	280	1387	1085	49
+1	281	1376	1101	50
+1	282	1302	1195	50
+1	283	1287	1217	50
+1	284	1223	1246	50
+1	285	1223	1256	50
+1	286	1163	1251	50
+1	287	1244	1291	50
+1	288	1249	1298	50
+1	289	1441	1087	50
+1	290	1373	1152	50
+1	291	1260	1176	50
+1	292	1281	1220	50
+1	293	1197	1310	50
+1	294	1308	1240	50
+1	295	1336	1141	49
+1	296	1332	1132	50
+1	297	1154	1304	50
+1	298	1326	1126	50
+1	299	1206	1278	50
+1	300	1273	1268	50
+1	301	1265	1188	50
+1	302	1239	1291	50
+1	303	1224	1236	50
+1	304	1231	1332	50
+1	305	1285	1214	50
+1	306	1342	1067	50
+1	307	1300	1199	50
+1	308	1400	1221	50
+1	309	1210	1221	50
+1	310	1399	1157	50
+1	311	1170	1312	50
+1	312	1321	1165	50
+1	313	1379	1167	50
+1	314	1162	1355	50
+1	315	1226	1251	50
+1	316	1370	1245	50
+1	317	1314	1094	50
+1	318	1325	1229	50
+1	319	1348	1172	50
+1	320	1466	1130	50
+1	321	1340	1185	50
+1	322	1268	1189	50
+1	323	1312	1142	50
+1	324	1279	1291	50
+1	325	1331	1281	50
+1	326	1299	1229	50
+1	327	1363	1169	50
+1	328	1217	1233	50
+1	329	1365	1168	50
+1	330	1188	1269	50
+1	331	1314	1188	50
+1	332	1289	1251	50
+1	333	1116	1395	50
+1	334	1224	1262	50
+1	335	1311	1173	50
+1	336	1479	1087	50
+1	337	1309	1132	50
+1	338	1096	1381	50
+1	339	1167	1340	50
+1	340	1405	1083	50
+1	341	1363	1155	50
+1	342	1408	1031	50
+1	343	1301	1286	50
+1	344	1218	1297	50
+1	345	1296	1239	50
+1	346	1255	1255	50
+1	347	1400	1157	50
+1	348	1302	1238	50
+1	349	1244	1216	50
+1	350	1247	1266	50
+1	351	1372	1144	50
+1	352	1257	1192	50
+1	353	1262	1228	50
+1	354	1490	1057	50
+1	355	1141	1382	49
```

Comment:
Good catch.  This is because in the python script that simulated this pon there is an a read-count threshold---it doesn't include a sample if the read counts it has at the site are below the threshold.

This `NUM_SAMPLES` column isn't currently used, so I'll remove it from the test files.  However, it might actually be good to store this in future versions of the allelic pon (in case someone is interested in seeing how many samples actually contained each site).  I'll file an issue for this.